tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrAllowed.hlsl

```diff
@@ -0,0 +1,15 @@
+// RUN: %dxc -E main -T lib_6_7 %s | FileCheck %s
```

Comment:
library compiles don't require entry points. To ensure that this is succeeding not due to it being an entry point, it would be best to eliminate the -E and rename the function too since "main" is the default entry point .